core/Application.php

```diff
@@ -163,6 +163,13 @@ function (GenericEvent $event) use ($container) {
 						$subject->addHintForMissingSubject($table->getName(), 'schedulobj_principuri_index');
 					}
 				}
+
+				if ($schema->hasTable('oc_properties')) {
+					$table = $schema->getTable('oc_properties');
```

Comment:
```suggestion
				if ($schema->hasTable('properties')) {
					$table = $schema->getTable('properties');
```

The prefix is added later (because it could be the non-default one ;))